scripts/functions/requirements/osx_brew

```diff
@@ -314,24 +314,24 @@ requirements_osx_brew_define_openssl()
   undesired_check openssl098
 
   case "$1" in
-    (ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.9*|ruby-1.8*)
-      brew_openssl_package="openssl"
+    (ruby-3.1*|ruby-3.0*|ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.9*|ruby-1.8*)
+      brew_openssl_package="openssl@1.1"
       ;;
 
     (ree-1.8*)
-      brew_openssl_package="openssl"
+      brew_openssl_package="openssl@1.1"
       ;;
 
     (rbx-1*|rbx-2*)
-      brew_openssl_package="openssl"
+      brew_openssl_package="openssl@1.1"
       ;;
 
     (truffleruby*)
```

Comment:
truffleruby openssl support has also been updated recently and I'm wondering if this switch case also needs to be expanded for specific truffle ruby versions 🤔 

https://github.com/oracle/truffleruby/issues/2660#issuecomment-1158058741